ls1intum/Artemis
# `Development`: Add adaptive learning as a documented feature

### Checklist
#### General



### Motivation and Context



@MaximilianAnzinger will work on Adaptive Learning and competencies. Therefore, we should introduce a feature tag for it and give it a separate documentation page.

### Description


- Extract documentation about competencies from the learning analytics page onto a dedicated page
- Add Adaptive learning as feature on the about us page and the README
- add Adaptive learning as a new feature tag for GitHub

### Steps for Testing
Code review.

- Check the new README file
- Check the updated documentation, available at https://artemis-platform--6798.org.readthedocs.build/en/6798/

### Review Progress

#### Code Review
